sdk/python/flet/alert_dialog.py

```diff
@@ -61,26 +61,27 @@ def open_dlg_modal(e):
 
     Online docs: https://flet.dev/docs/controls/alertdialog
     """
+
     def __init__(
-        self,
-        ref: Optional[Ref] = None,
-        disabled: Optional[bool] = None,
-        visible: Optional[bool] = None,
-        data: Any = None,
-        #
-        # Specific
-        #
-        open: bool = False,
-        modal: bool = False,
-        title: Optional[Control] = None,
-        title_padding: PaddingValue = None,
-        content: Optional[Control] = None,
-        content_padding: PaddingValue = None,
-        actions: Optional[List[Control]] = None,
-        actions_padding: PaddingValue = None,
-        actions_alignment: MainAxisAlignment = MainAxisAlignment.NONE,
-        shape: Optional[OutlinedBorder] = None,
-        on_dismiss=None,
+            self,
```

Comment:
Something has happened to an indentation here. Could you please check/fix?

Reply:
Fixed in next commit.